syndesis/tests/org.fusesource.ide.syndesis.extension.ui.tests.integration/src/main/java/org/fusesource/ide/syndesis/extensions/tests/integration/SyndesisExtensionProjectCreatorRunnableIT.java

```diff
@@ -76,20 +76,25 @@ public void setup() throws WorkbenchException {
 		// TODO: for now we need the staging repos, disable before GA
 		new StagingRepositoriesPreferenceInitializer().setStagingRepositoriesEnablement(true);
 
+		versions = new IgniteVersionMapper().getMapping();
+		
 		SyndesisExtensionIntegrationTestsActivator.pluginLog()
 				.logInfo("End setup for " + SyndesisExtensionProjectCreatorRunnableIT.class.getSimpleName());
 	}
 
-	private SyndesisExtension createDefaultNewSyndesisExtension() {
+	protected abstract AbstractProjectTemplate getTemplate();
+	
+	protected String getCamelResourcePath() {
+		return null;
+	}
+	
+	protected SyndesisExtension createDefaultNewSyndesisExtension() {
 		SyndesisExtension extension = new SyndesisExtension();
-		SyndesisExtensionsUtil.IgniteVersionInfoModel model = SyndesisExtensionsUtil.getIgniteVersionModel();
-		extension.setSpringBootVersion(model.getSpringBootVersion());
-		extension.setCamelVersion(model.getCamelVersion());
-		extension.setSyndesisVersion(model.getSyndesisVersion());
+		extension.setSyndesisVersion(versions.keySet().toArray()[versions.size()-1].toString());
```

Comment:
we take whatever version available?
Not choose the latest or the first?
done like that because there is a single one for now?

Reply:
You are right. I should have sorted the resultset and take the latest version. For now it won't matter as there is just one version available. Will add it.